tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/03_productsStock/06_enableDeliveryTimeInStockProducts.js

```diff
@@ -0,0 +1,99 @@
+require('module-alias/register');
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_productSettings_productsStock_enableDeliveryTimeInStockProducts';
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ProductSettingsPage = require('@pages/BO/shopParameters/productSettings');
+const FOProductPage = require('@pages/FO/product');
+const FOHomePage = require('@pages/FO/home');
+
+let browser;
+let page;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    productSettingsPage: new ProductSettingsPage(page),
+    foProductPage: new FOProductPage(page),
+    foHomePage: new FOHomePage(page),
+  };
+};
+
+describe('Enable/Disable enable stock management', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO
+  loginCommon.loginBO();
+
+  it('should go to \'Shop parameters > Product Settings\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToProductSettingsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.shopParametersParentLink,
+      this.pageObjects.boBasePage.productSettingsLink,
+    );
+    const pageTitle = await this.pageObjects.productSettingsPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.productSettingsPage.pageTitle);
+  });
+
+  const tests = [
+    {args: {action: 'enable', enable: true, deliveryTimeText: '3-4 days'}},
```

Comment:
```suggestion
    {args: {enable: true, deliveryTimeText: '3-4 days'}},
```

Reply:
why do you want to delete it,  this is used to know which action is done before the check. and without it, we can't know if the checks are after enabling or disabling